fawkes/constants/constants.py

```diff
@@ -123,3 +126,11 @@
 WEEKLY_EMAIL_TEMPLATE = "fawkes/email_summary/templates/weekly.html"
 WEEKLY_EMAIL_DETAILED_TEMPLATE = "fawkes/email_summary/templates/weekly-detailed.html"
 WEEKLY_EMAIL_DETAILED_REVIEW_BLOCK_TEMPLATE = "fawkes/email_summary/templates/weekly-detailed-review-block.html"
+
+#Summrization constants
+num_clusters = 2
+# num_result_per_review=2
+minimum_reviews_per_cluster=10 # OR based on similarity in a cluster ( only retun in high rating)
+max_words_in_sentence= 20
+min_words_in_sentence= 4
+summary_length_per_cluster=30
```

Comment:
Why are all these configs defined here ? Can these be user-configurable ?

Reply:
These are configs for the code, for eg. max_words_in_sentence= 20, min_words_in_sentence= 4 are specific to the code. The user should ideally have nothing to do with these